timber/src/main/java/timber/log/Timber.java

```diff
@@ -446,13 +455,11 @@ protected String createStackElementTag(StackTraceElement element) {
      *
      * {@inheritDoc}
      */
-    @Override protected void log(int priority, String tag, String message, Throwable t) {
+    @Override protected void log(int priority, String tag, String message, Throwable e) {
+      String callerInfo = getCallerInfo(new Throwable().getStackTrace());
```

Comment:
It's better if there was a way to share stack trace with `log()` and `getTag()`; but I think it's not important because it's a debug feature anyway.